requirements.txt

```diff
@@ -32,3 +32,4 @@ netifaces==0.10.4
 pyjokes==0.5.0
 psutil==4.1.0
 pep8==1.7.0
+-e git+https://github.com/ChristopherRogers1991/phue#egg=phue
```

Comment:
why are we depending on your fork of phue? And if we're going to depend on something via git, it has to be at a tag or sha (pulling from HEAD means that every build gets a potentially different version, which is bad).